### PR TITLE
Polish language update after Review

### PR DESCRIPTION
**Pull request by Triloworld**
It's update of Polish language after Review #1094 
More polished version of existing one.

### CODE REVIEW

**Comment by Kun2210** [comment]
Thanks for your comments, appreciated. On the 'Playback API' translation - if we're going to go with 'Dostęp do API', then there's one more line that needs an update:

`"modal-confirmOpenPort-content": "Playback API nie mogło zostać uruchomione, ponieważ jest ono blokowane przez Windows Firewall. Możemy to dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",`

So, following your way of thinking, I'd suggest something like:

`"modal-confirmOpenPort-content": "Nie można udzielić dostępu do API, ponieważ operacja została zablokowana przez Windows Firewall. Możemy go dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",`

Let me know your thoughts on this.

**Comment by Triloworld** [comment]
Thanks @Kun2210 for pointing that 👍 I will update it. One more think. Change Windows to just firewall. It can be even custom firewall (not built in system) and of course in other OS it will silly to read about Windows ;)

**Comment by Triloworld** [comment]
There is one more think. I don't know whois doing that but PL language for now isn't working correctly. In 
`const localeLinks = {
  'en-029': 'en-US',
  'en-AU': 'en-US',
  'en-BZ': 'en-US',
  'en-CA': 'en-US',
  'en-GB': 'en-US',
  'en-IE': 'en-US',
  'en-IN': 'en-US',
  'en-JM': 'en-US',
  'en-MY': 'en-US',
  'en-NZ': 'en-US',
  'en-PH': 'en-US',
  'en-SG': 'en-US',
  'en-TT': 'en-US',
  'en-ZA': 'en-US',
  'en-ZW': 'en-US',
  'pt-BR': 'pt-BR',
  'pt-PT': 'pt-BR',
};`
There isn't pl-PL locale. Add one? File of Polish language is there but app not switch automatically to it.

**Comment by Kun2210** [comment]
@Triloworld 
On the firewall thing - actually the original en-US file points directly to Windows Firewall. So I believe this message might be displayed only on Windows OS. @MarshallOfSound can you please confirm that?

Also @MarshallOfSound, do we need to add pl-PL to `const localeLinks` for the Polish language to work?

**Comment by MarshallOfSound** [comment]
> On the firewall thing - actually the original en-US file points directly to Windows Firewall. So I believe this message might be displayed on Windows OS only. @MarshallOfSound can you please confirm that?

The firewall prompt will only appear on Windows

> Also @MarshallOfSound, do we need to add pl-PL to const localeLinks for the Polish language to work?

No you don't.  That object is for aliasing language codes to other language codes

**Comment by Kun2210** [comment]
@MarshallOfSound thanks for clearing this out.
@Triloworld I would then leave Windows Firewall as is, since it seems like the blocking issue is related only with WF.

**Comment by MarshallOfSound** [comment]
Can I get a Go / No Go on this PR.

Not sure if it's ready

**Comment by Triloworld** [comment]
@Kun2210 Yes ok. It's fine ;) Thanks. For me is a Go like say @MarshallOfSound 
And @Kun2210 it's Go state?

**Comment by Kun2210** [comment]
Seems OK to me.

File: src/_locales/pl-PL.json [hunk 1]
```diff
@@ -42,9 +42,9 @@
 
   "modal-confirmTray-title": "Chcieliśmy tylko uprzedzić",
   "modal-confirmTray-content": "Zamknięcie głównego okna aplikacji (czyli to, co właśnie próbowałeś zrobić) w rzeczywistości zminimalizuje odtwarzacz do systemowego paska narzędzi. Możesz zmienić to ustawienie w oknie Ustawień Aplikacji.",
-  "modal-confirmOpenPort-content": "Playback API nie mogło zostać uruchomione, ponieważ jest ono blokowane przez Windows Firewall. Możemy to dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",
+  "modal-confirmOpenPort-content": "Nie można udzielić dostępu do API, ponieważ operacja została zablokowana przez Windows Firewall. Możemy go dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",
   "modal-confirmUpdate-title": "Dostępna aktualizacja",
-  "modal-confirmUpdate-content": "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy zaktualizowanie aplikacji najszybciej jak to możliwe, żeby zapewnić najlepsze wrażenia z korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!",
```
Comment (MarshallOfSound) [inline src/_locales/pl-PL.json]: Woah

Almost let this one slip through.  The `<br />` is required and must remain in this string

**Comment by Triloworld** [comment]
@MarshallOfSound Thanks for help because it is my second time when creating pull request so glad you are patient enough 👍.

**Comment by Kun2210** [comment]
@Triloworld 
There's a new string to be translated:
"settings-option-enable-taskbar-progress": "",

I guess we can incorporate that into your pull request and then finally merge it into master. I suggest:
"Pokaż postęp utwór na pasku systemowym", though I'm not sure if it sounds well. Any other ideas?

**Comment by Triloworld** [comment]
Sorry for delay. That sugession have some problems. With correction it will be: "Pokazuj postęp utworu na pasku systemowym".

**Comment by Triloworld** [comment]
Ok. That would be last one think. I'am ok with It right now and if @Kun2210 is ok we can merge it :)

**Comment by Kun2210** [comment]
I'm OK with that.

**Pull request merged**